oioswift/proxy/controllers/account.py

```diff
@@ -17,7 +17,7 @@
 import time
 from xml.sax import saxutils
 
-from swift.common.request_helpers import get_listing_content_type
+from swift.common.middleware.listing_formats import get_listing_content_type
```

Comment:
It's not compatible with Pike: https://github.com/openstack/swift/commit/4806434cb0e857ce624c62df2a262e3c3bb9f4d1